neo/rawio/maxwellrawio.py

```diff
@@ -91,7 +91,17 @@ def _parse_header(self):
         for stream_id in signal_streams['id']:
             if int(version) == 20160704:
                 sr = 20000.
-                gain_uV = h5['settings']['lsb'][0] * 1e6
+                settings = h5["settings"]
+                if 'lsb' in settings:
+                    gain_uV = settings['lsb'][:][0] * 1e6
```

Comment:
any reason for the additional `[:]`?
```suggestion
                    gain_uV = settings['lsb'][0] * 1e6
```

Reply:
@JuliaSprenger no, but a difference between the 2 versions was already there (https://github.com/NeuralEnsemble/python-neo/blob/master/neo/rawio/maxwellrawio.py#L103). Should we stick to the `settings['lsb'][0]` notation for all?

Reply:
If it's working in all cases then I would argue to simplify the code in all places.